indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -70,5 +70,29 @@
     </div>
     <script>
         $('.js-edit-tpl-dropdown').parent().dropdown();
+        $('.email-template-list').sortable({
+            axis: 'y',
+            containment: 'parent',
+            cursor: 'move',
+            distance: 2,
+            handle: '.ui-i-box-sortable-handle',
+            items: '> li.email-template',
+            tolerance: 'pointer',
+            forcePlaceholderSize: true,
+            update: function(e) {
+                var $elem = $('.email-template-list');
+                var sortedList = _.map($elem.find('li.email-template'), function(elem) {
```

Comment:
jQuery always returns a jQuery object (which in the end is just an array on steroids) from `.map()` even if the callback doesn't return DOM objects but something else